docs/explanation/tutorial-guidelines.md

```diff
@@ -0,0 +1,9 @@
+This document describes a checklist for writing tutorials.
+
+- Write in the second person (i.e., "you" instead of "we").
+- Start with a high-level conceptual overview of what the user will accomplish.
+- There should a "Prerequisites" section that lists the software requirements as well as other tutorials that the user should complete prior to starting the current tutorial.
```

Comment:
```suggestion
- There should be a "Prerequisites" section that lists the software requirements as well as other tutorials that the user should complete prior to starting the current tutorial.
```

Reply:
this makes it sound like every tutorial should say to make sure you have python installed in the prereqs...

```suggestion
- There should be a "Prerequisites" section that links to tutorials that the user should complete prior to starting the current tutorial as well as any additional software needed for this tutorial
```